packages/matchbox/src/components/Button/Group.tsx

```diff
@@ -4,11 +4,16 @@ import styled from 'styled-components';
 import { Box } from '../Box';
 import { group } from './styles';
 
+type GroupProps = {
+  className?: string;
+  children?: React.ReactNode;
+};
+
 const StyledGroup = styled(Box)`
   ${group}
 `;
 
-const Group = ({ children, className = '' }) => (
+const Group: React.FC<GroupProps> = ({ children, className = '' }) => (
```

Comment:
I don't think we need React.FC here since you're explicitly typing className and children